### ISSUE TITLE
Audio stutters on Android Firefox

### ISSUE THREAD

**Issue by Wyzmic** [yomidevs/yomitan#1337]
I tried it on kiwi browser and there's no issues

**Browser version**
129.0.2

**Yomitan version**
24.8.12.1

https://github.com/user-attachments/assets/c2e1526e-aaa8-47ee-a09e-5b3c259baaa4

**Comment by Lolle2000la** [comment]
Can confirm, am having this problem too. It does not occur on Firefox for Windows.

It happens especially at the beginning and cuts of at the end and in some cases creates a loud popping sound. For longer samples it correctly plays the middle, as if there is some kind of wind up that needs to be done.

**Comment by Wyzmic** [comment]
> Can confirm, am having this problem too on Firefox for Android. It does not occur on Firefox for Windows.
> 
> It happens especially at the beginning and cuts of at the end and in some cases creates a loud popping sound. For longer samples it correctly plays the middle, as if there is some kind of wind up that needs to be done.

Yeah exactly, as of the latest firefox and yomitan version it still occurs.

**Comment by Wyzmic** [comment]
It's fixed! Not sure if the latest yomitan release or firefox fixed this issue but I can properly play audio now
